### PR TITLE
[CAMEL-21451] Migrate to JKube generated resources for all runtimes

### PR DESCRIPTION
**Pull request by tdiesler**
https://issues.apache.org/jira/browse/CAMEL-21451

### CODE REVIEW

**Review by davsclaus — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

File: dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesExport.java [hunk 1]
```diff
@@ -187,20 +186,17 @@ public Integer export() throws Exception {
             propPrefix = runtime.runtime();
         }
 
-        boolean useQuarkusPlugin = runtime == RuntimeType.quarkus;
-        boolean useJKubePlugin = !useQuarkusPlugin;
+        boolean isQuarkusRuntim = runtime == RuntimeType.quarkus;
```
Comment (gansheer) [inline dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesExport.java:189]:
```suggestion
        boolean isQuarkusRuntime = runtime == RuntimeType.quarkus;
```

**Review by gansheer — CHANGES_REQUESTED** [review]
If you remove the quarkus way of generating the manifests then this code should be removed as well :
https://github.com/apache/camel/blob/9df896a52b666143c292db804e28c1c8bb2b31c9/dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/traits/RouteTrait.java#L123-L170

**Comment by tdiesler** [comment]
please test this again

**Review by gansheer — APPROVED** [review]
LGTM.

**Pull request merged**